content/api/commands/as.md

```diff
@@ -46,8 +46,8 @@ using an `@` prefix.
 
 ### Yields [<Icon name="question-circle"/>](/guides/core-concepts/introduction-to-cypress#Subject-Management)
 
-<List><li>`.as()` yields the same subject it was given from the previous
-command.</li></List>
+- `.as()` yields the same subject it was given.
+- `.as()` is a query, and it _safe_ to chain further methods.
```

Comment:
It's a query because it's implemented with `Commands.addQuery()`, not `Commands.add()`. It's implemented this way so that it's safe to chain off of.

```
cy.get('button').as('btn').click()
```

is something we want to support, so `.as()` is a query. Basically, if something *can* be a query, it *should* be, to give our users maximum safety and freedom to string together methods as they please.

I think that's the basic intuition I'm having trouble conveying: "Queryness" is about *how methods chain together*, not about the DOM. Very open to a better name, if you have any suggestions.

Reply:
there seems to be a number of "utility" functions that fall into scenario. I think calling them queries is confusing. Maybe we can refer to them as utilities (also open to better naming), and call out they are chainable?

Reply:
Utility is heavily overloaded already - we have 'utility commands', and a 'Utilities' section in the API docs.

How about just removing the word "query" here?

```
- It is _safe_ to chain further methods after `.as()`.
```

Reply:
The three sticking points for the query name seem to be:
- .as()
- .invoke()
- .its()

I'd argue that .invoke() and .its() fit very well.

"I to know this button's title" -> cy.get('button').invoke('attr', 'title')
"I want to know the value of this form" -> cy.get('form').invoke('val')
"I want to know the request body" -> cy.get('@myRequest.1').its('req.body')
"I want to know the user's name" -> cy.wrap(userObj).its('name.first')

All of these are very naturally thought of as "queries". In SQL, you query the DOM. In CSS, you query the DOM. In Cypress, you query your application - Cypress allows subjects other than DOM elements, so you should be able to query things other than DOM elements.

Reply:
We discussed in a meeting an decided that we'd change things around a bit into a hierarchy:

Commands:
- Queries
- Actions
- Assertions
- ...other commands...

I'm updating the PR to reflect these changes.